BinaryEncoding.md

```diff
@@ -390,10 +387,10 @@ out of range, `br_table` branches to the default target.
 
 The `memory_immediate` type is encoded as follows:
 
-| Name | Type | Present? | Description |
-| ---- | ---- | ---- | ---- |
-| flags | `uint8` | always | a bitfield where<br>bit `4` indicates an offset follows<br>bit `7` indicates natural alignment<br>other bits are reserved for future use |
-| offset | `varuint32` | `flags[4] == 1` | the value of the offset |
+| Name | Type | Description |
+| ---- | ---- | ---- |
+| offset | `varuint32` | the value of the offset |
```

Comment:
As I suggested in IRC, maybe swap the order of offset and align? If we assume that the memory access is often naturally aligned, then it should be better for compression (using nullary macros or otherwise) paired with the memory access opcode that precedes it.